python/ray/rllib/models/catalog.py

```diff
@@ -15,10 +15,13 @@
 MODEL_CONFIGS = [
     "conv_filters",
     "downscale_factor",
+    "grayscale",
+    "zero_mean",
     "extra_frameskip",
     "fcnet_activation",
     "fcnet_hiddens",
-    "free_log_std"
+    "free_log_std",
+    "use_lstm"
```

Comment:
Since this isn't actually supported except for a3c, move it to the a3c config for now?

Reply:
Unfortunately we can't just keep it in a3c config due to https://github.com/richardliaw/ray/blob/cf8ba457c0201431ce158aeccc730e33f34ba167/python/ray/rllib/models/catalog.py#L100